src/primitives/circle.jl

```diff
@@ -14,6 +14,29 @@ struct Circle{T} <: Primitive{3,T}
   radius::T
 end
 
+"""
+    Circle(p1, p2, p3)
+
+A 3D circle passing through points `p1`, `p2` and `p3`.
```

Comment:
Do we have a good reference for this method?

Reply:
No idea. It comes from my old archives.

Reply:
No problem 👍🏽